packages/common-sdk/src/web3/token-util.ts

```diff
@@ -109,7 +109,9 @@ export class TokenUtil {
       tokenMint,
       getAccountRentExempt,
       amount,
-      payer
+      payer,
+      undefined,
+      allowPDASourceWallet
```

Comment:
Is it comfortable to use ``true`` instead of ``allowPDASourceWallet`` because it is not source and I think it is always okay to use PDA as destination owner(wallet) if dev want.

Reply:
From my personal pov yeah using true by default is always good 😁
Depending what you prefer here I can clearly change it to just use true directly!

Reply:
@bguillaumat 
thanks for the reply.
I believe now we agreed to use ``true``, could you update the PR.

Reply:
Updated

Reply:
Thanks for the update.
``allowPDASourceWallet`` is for ``Source``, so it is used to derive source, but not for dest.
So I think this change doesn't break anything. 🙂